### ISSUE TITLE
Allow users to choose between the stable and git master versions of Codespell

### ISSUE THREAD

**Issue by peternewman** [codespell-project/actions-codespell#5]
Looks like it should be possible by passing args into the Dockerfile
https://help.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file
And here:
https://docs.docker.com/engine/reference/builder/

**Comment by cclauss** [comment]
Is this task complete?

**Comment by peternewman** [comment]
> Is this task complete?

I don't believe so, we still just load them in via requirements.txt which specifies a tagged version currently:
https://github.com/codespell-project/actions-codespell/blob/master/requirements.txt

**Comment by cclauss** [comment]
<img width="336" alt="image" src="https://user-images.githubusercontent.com/3709715/228729780-fb97e7c6-b818-4031-a5b7-5b9adb8b1b0e.png">

**Comment by cclauss** [comment]
- uses: codespell-project/actions-codespell@v1
- uses: codespell-project/actions-codespell@stable
- uses: codespell-project/actions-codespell@master
